netcdf/attributes.go

```diff
@@ -40,6 +45,22 @@ func (a Attr) Len() (n uint64, err error) {
 	return
 }
 
+// ValueString returns the value of the attribute value of type text.
+func (a Attr) ValueString() (val string, err error) {
```

Comment:
In my original comment, I meant we don't need this function, since it's very easy to convert a []byte to a string. So, remove it.